backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -348,6 +361,9 @@ public UserInfo resendActivationEmail(UUID userId) {
             .orElseThrow(MisconfiguredUserException::new);
     Organization org = _orgService.getOrganization(orgClaims.getOrganizationExternalId());
     OrganizationRoles orgRoles = _orgService.getOrganizationRoles(org, orgClaims);
+    if (!_featureFlagsConfig.isOktaMigrationEnabled()) {
+      setRolesAndFacilities(orgRoles, apiUser);
```

Comment:
is it the case that we'll need to add this set function wherever there's call to `getOrganizationRoles` that also involves a grab from the api user? If so, wonder if we want to be more systematic to solve this at once rather than adding this snippet each time. feel like I don't have enough context to know whether that'd be worth it configuring we might want to more properly refactor the update flow once more of the code is written, but would love thoughts!

Reply:
Aren't these particular snippets just temporary during this phase of the migration? I think it's alright to leave them since it feels more readable this way compared to creating an abstraction that will later be removed